dist/profile/manifests/buildmaster.pp

```diff
@@ -33,10 +34,21 @@
   $groovy_d_lock_down_jenkins      = 'absent',
   $groovy_d_terraform_credentials  = 'absent',
   $jcasc_configs                   = [],
+  $jcasc_reload_token                     = '',
```

Comment:
indenting looks off here, I won't use a suggestion as i'll probably get the spaces wrong

Reply:
Sounds like it's my IDE's Puppet formatter which is confused by the comment on the next line. Weird. Fixed, thanks!